src/main/java/svc/data/citations/datasources/tyler/TylerCitationDataSource.java

```diff
@@ -1,59 +1,106 @@
 package svc.data.citations.datasources.tyler;
```

Comment:
I know there is an integration test and that's fine for now but we probably need a unit test for this at some point too